hash/apple2gs_flop_orig.xml

```diff
@@ -3482,4 +3482,119 @@ license:CC0
+		<year>1987</year>
+		<publisher>Sierra On-Line</publisher>
+		<info name="release" value="2022-03-13"/>
+		<sharedfeat name="compatibility" value="A2GS"/>
+		<!-- It requires a 512K Apple IIgs. -->
+		<!--"Mixed-Up Mother Goose" is a 1987 adventure game designed by Roberta Williams, with graphics by Jerry Moore, programming by David Slayback, music by Amenda-Lombardo, and distributed by Sierra On-Line. This is version 2.0A. It requires a 512K Apple IIgs.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1"/>
+			<dataarea name="flop" size="1296242">
+				<rom name="mixed up mother goose iigs disk 1.woz" size="1296242" crc="d20296b0" sha1="38f0d20f2c4e444dd6360d4a2f3786a812d87156"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2"/>
+			<dataarea name="flop" size="1296242">
+				<rom name="mixed up mother goose iigs disk 2.woz" size="1296242" crc="17e29c48" sha1="53990f6e6d74da67e630057d69ad85bbe8c86155"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="gnargolf">
+		<description>Gnarly Golf</description>
+		<year>1989</year>
+		<publisher>Britannica Software</publisher>
+		<info name="release" value="2022-03-16"/>
+		<sharedfeat name="compatibility" value="A2GS"/>
+		<!-- It requires a 512K Apple IIgs. -->
+		<!--"Gnarly Golf" is a 1989 sports game developed by Jim Coliz Jr., Darren Bartlett, Andy Hildebrand, and Greg Thomas, and distributed by Britannica Software. It requires a 512K Apple IIgs.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1 - Program disk"/>
+			<dataarea name="flop" size="1296181">
+				<rom name="gnarly golf iigs disk 1 - program disk.woz" size="1296181" crc="54df53f2" sha1="a755251a853eb7beb826151b8b51b19ef879d814"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2 - Course disk"/>
+			<dataarea name="flop" size="1296181">
+				<rom name="gnarly golf iigs disk 2 - course disk.woz" size="1296181" crc="d4e59bf4" sha1="7a5ee6acbfe98170c3f42b214a71ee26d9043e33"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="4thinchs">
+		<description>4th and Inches</description>
+		<year>1988</year>
+		<publisher>Accolade</publisher>
+		<info name="release" value="2022-03-06"/>
+		<sharedfeat name="compatibility" value="A2GS"/>
+		<!-- It requires a 512K Apple IIgs. -->
+		<!--"4th and Inches" is a 1988 sports program developed by Tony Manso, with music by Ed Bogas, artwork by Les Pardew, and distributed by Accolade. It requires a 512K Apple IIgs.-->
+		<!--"4th and Inches Team Construction Disk" is a 1988 sports program developed by Tony Manso and distributed by Accolade. It requires a 512K Apple IIgs and the main "4th and Inches" disk (&lt;a href="/details/wozaday_4th_and_Inches_IIgs" rel="nofollow"&gt;sold separately&lt;/a&gt;).-->
```

Comment:
End of this comment looks like a copy/paste error.

Reply:
No copy/paste error. 
I merged, much like the long-preexisting apple2gs_flop_misc version, the game and the utility disk into one entry.

Reply:
I think the point is more that there are HTML entities in the comment.